pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminBuilderImpl.java

```diff
@@ -44,6 +46,8 @@
 
     @Override
     public PulsarAdmin build() throws PulsarClientException {
+        checkArgument(StringUtils.isNotBlank(conf.getServiceUrl()), "Service URL needs to be specified");
```

Comment:
Ehy mot oushing this check inside the constructor of PulsarAdminImpl?

Reply:
The `ClientBuilder` also does the same way. https://github.com/apache/pulsar/blob/5ebaafd4288c69d135b295ab563c8ad56affa15b/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java#L53

Reply:
+1 

It's better to move to the constructor of PulsarAdminImpl.
We can change the ClientBuilderImpl.java

Reply:
@codelipenghui @eolivelli  I have moved this argument check to the constructor. PTAL. Thanks.